e2e/libs/volume/rest.py

```diff
@@ -15,19 +15,13 @@
 class Rest(Base):
 
     def __init__(self, node_exec):
-        self.longhorn_client = common_utils.get_longhorn_client()
+        self.longhorn_client = Utility().get_longhorn_client()
```

Comment:
I think methods in utility is better to be flat functions instead of be part of Utility class. No need to have everything wrapped in a Class if it's not appropriate. 

Here, we can just import `get_longhorn_client` from the utility module, then use it directly.

Reply:
Modified in PR https://github.com/longhorn/longhorn-tests/pull/1520